### PR TITLE
Update version to 1.0.0

### LINKED ISSUE #207: Invalid relative path concatenation in get_chunk_url
**Context:**

I am using django-webpack-loader in a multi-pages Django app that runs VueJS components in Django templates. I set the publicPath option in webpack to an empty string so that I can control the static url from the Django settings and properly use a CDN server, as described in #160

**Issue:**

JS and CSS assets are saved to css and js subfolders. Here is what _webpack-stats.json_ looks like:

```{
    "status": "done",
    "chunks": {
        "compA": [{
            "name": "css/compA.css",
            "path": "D:\\home\\site\\wwwroot\\common-static\\assets\\frontend\\css\\compA.css"
        }, {
            "name": "css/compA.3564e3ce75ddc57936fd.css",
            "path": "D:\\home\\site\\wwwroot\\common-static\\assets\\frontend\\css\\compA.3564e3ce75ddc57936fd.css"
        }, 
             ...
        }],
             ...
}
```
Note that the chunk name uses a UNIX-like path "css/compA.css". 

In _settings.py_ I have the following configuration:
```
WEBPACK_LOADER = {
    'DEFAULT': {
        # when true, webpack will read stats file only once and cache results
        'CACHE': not DEBUG,
        # dir in which webpack outputs the bundles - it should not be full path
        'BUNDLE_DIR_NAME': os.path.join("assets", "frontend", ""),  # add a trailing slash
        # location of the bundle tracker file
        'STATS_FILE': STATS_FILE,
        'POLL_INTERVAL': 0.3,
        'TIMEOUT': None,
        'IGNORE': [r'.+\.hot-update.js', r'.+\.map']
    }
}
```
The BUNDLE_DIR_NAME value ends up being 'assets\\frontend\\' on Windows. 

With these settings, django-webpack-loader prints an invalid path in the Django template by concatenating BUNDLE_DIR_NAME with the chunk name: **\static\assets\frontend\css/compA.css**

The issue is in the get_chunk_url of **webpack_loader/loader.py**:

```
def get_chunk_url(self, chunk):
        public_path = chunk.get('publicPath')
        if public_path:
            return public_path

        relpath = '{0}{1}'.format(
            self.config['BUNDLE_DIR_NAME'], chunk['name']
        )
        return staticfiles_storage.url(relpath)
```
Concatenation does not take into account that the two relative paths may be formatted differently. In addition it requires the BUNDLE_DIR_NAME settings to end with a slash.

**Solution**

This issue is easily fixed by replacing the path concatenation with:
```
def get_chunk_url(self, chunk):
        public_path = chunk.get('publicPath')
        if public_path:
            return public_path

        relpath = os.path.normpath(
            os.path.join(self.config['BUNDLE_DIR_NAME'], chunk['name']
        )
        return staticfiles_storage.url(relpath)
```
This makes the path concatenation OS-independent, and allow greater flexibility to define output paths in the webpack config.

### LINKED ISSUE #212: Package Needs Updates
The django community is very dependent on this package, however, it's now outdated enough that it no longer works according to the documentation. Can the maintainer please add some other admins onto this project so we can get it working again?

If not, I'm happy to fork it and start accepting pull requests for fixes.

### LINKED ISSUE #215: 1.0.0
Starting this issue to track what should go into 1.0.0. This will be a backwards incompatible release so we have a chance to improve things we couldn't before.  For one, 1.0.0 will only support webpack 4 and newer. 

Please add a comment or link to an issue if you'd like it to be resolved in 1.0.0

### LINKED ISSUE #253: What's the status of this repo?
Looking through current issues, it looks like the project is moving over to Jazzband and will continue to be maintained. But there is also this comment: https://github.com/owais/django-webpack-loader/issues/238#issuecomment-705269647, that says this project is no longer maintained.  If this project is no longer maintained, could we add that to the README, just to make it more clear?

### PR DESCRIPTION
**Pull request by rvlb**
Closes #207 
Closes #212 
Closes #215 
Closes #253

### CODE REVIEW

**Review by fjsj — CHANGES_REQUESTED** [review]
@rvlb please check the issues on milestone 1.0.0: https://github.com/django-webpack/django-webpack-loader/milestone/1

File: README.md [hunk 1]
```diff
@@ -23,7 +19,7 @@ A [changelog](CHANGELOG.md) is also available.
 
 ## Compatibility
 
-Test cases cover Django>=1.6 on Python 2.7 and Python>=3.4. 100% code coverage is the target so we can be sure everything works anytime. It should probably work on older version of django as well but the package does not ship any test cases for them.
+Test cases cover Django>=2.0 on Python>=3.5. 100% code coverage is the target so we can be sure everything works anytime. It should probably work on older version of django as well but the package does not ship any test cases for them.
```
Comment (fjsj) [inline README.md:22]: Can we add code coverage badges and supported version badges?

File: examples/hot-reload/README.md [hunk 2]
```diff
@@ -21,5 +21,3 @@ Run webpack dev server
 ```bash
 node server.js
 ```
-
-Now you can make changes to `assets/js/app.jsx` and the changes will show up in the browser automagically.
```
Comment (fjsj) [inline examples/hot-reload/README.md:25]: Any reason to remove this line?

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall looking nice. Some corrections TODO:

- We should add db.sqlite3 to .gitignore. This PR is adding `examples/simple/db.sqlite3`.
- Check inline comments.

File: webpack_loader/loader.py [hunk 3]
```diff
@@ -53,8 +54,9 @@ def get_chunk_url(self, chunk_file):
         if public_path:
             return public_path
 
-        relpath = '{0}{1}'.format(
-            self.config['BUNDLE_DIR_NAME'], chunk_file['name']
+        # Use os.path.normpath for Windows paths
+        relpath = os.path.normpath(
```
Comment (fjsj) [inline webpack_loader/loader.py:58]: Is this necessary?

Comment (rvlb) [reply webpack_loader/loader.py:58]: #207

**Review by fjsj — APPROVED** [review]
LGTM, thanks!

**Pull request merged**